src/Commands.pm

```diff
@@ -6098,4 +6099,47 @@ sub cmdDeadTime {
 	message $msg, "list";
 }
 
+sub cmdAchieve {
+	my (undef, $args) = @_;
+	my ($arg1) = $args =~ /^(\w+)/;
+	my ($arg2) = $args =~ /^\w+\s+(\S.*)/;
+	
+	if (($arg1 ne 'list' && $arg1 ne 'reward') || ($arg1 eq 'list' && defined $arg2) || ($arg1 eq 'reward' && !defined $arg2)) {
+		message "[eventMacro] Usage:\n".
+				"eventMacro include on <filename or pattern>\n".
+				"eventMacro include on all\n".
+				"eventMacro include off <filename or pattern>\n".
+				"eventMacro include off all\n".
+				"eventMacro include list\n", 'list';
+		return;
+	}
+
+	if ($arg1 eq 'reward') {
+		if (!exists $achievementList->{$arg2}) {
+			error "You don't have the achievement $arg2.\n";
```

Comment:
No translations? :D 🗡 
(lol I know this has already been long time under review :P)

@allanon are we cool with merging this? we can always improve it again later on (as long as it doesn't break backwards compatibility)